### ISSUE TITLE
Node scale up not working

### ISSUE THREAD

**Issue by aaryabhatt** [openshift/openshift-ansible#5263]
#### Description
I am trying to add more node servers in my 3 node cluster setup, one for master, one for infra and one for node. 

when I run this scale up file 
ansible-playbook playbooks/byo/openshift-node/scaleup.yml it gives the below error

> On a multi master install, if the first master goes down we can no
it is a single master and multiple node setup.

> longer scaleup the cluster with new nodes or masters.

##### Version

* Your ansible version per `ansible --version`
ansible-2.3.1.0-1.el7.noarch

If you're operating from a **git clone**:
yes

* The output of `git describe`
I run the git pull later now it is 
openshift-ansible-3.7.0-0.120.0-8-g6598350
still the same issue.

If you're running from playbooks installed via RPM or
`atomic-openshift-utils`
RPM based advanced installation

* The output of `rpm -q atomic-openshift-utils openshift-ansible`

Place the output between the code block below:

```
VERSION INFORMATION HERE PLEASE
oc version
oc v3.6.0+c4dd4cf
kubernetes v1.6.1+5115d708d7
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://prodmasternode1.example.com:8443
openshift v3.6.0+c4dd4cf
kubernetes v1.6.1+5115d708d7

```

##### Steps To Reproduce
ansible host inventory section add the below section for new_nodes

`[new_nodes]
prodwebnode2.example.com openshift_ip=136.243.80.24 openshift_public_ip=136.243.80.24 openshift_node_labels="{'region':'primary', 'zone':'east'}" openshift_public_hostname=prodwebnode2.example.com
prodwebnode3.example.com openshift_ip=136.243.80.27 openshift_public_ip=136.243.80.27 openshift_node_labels="{'region':'primary', 'zone':'east'}" openshift_public_hostname=prodwebnode3.example.com
`

then run the scale up playbook
##### Expected Results
Describe what you expected to happen.

```
It should add the nodes successfully.

```

##### Observed Results
Describe what is actually happening.

```
TASK [openshift_master_facts : Set master facts] *****************************************************************************************************
Tuesday 29 August 2017  23:12:03 -0700 (0:00:00.021)       0:00:01.698 ******** 
fatal: [prodmasternode1.groupzeal.com]: FAILED! => {
    "failed": true
}

MSG:

the field 'args' has an invalid value, which appears to include a variable that is undefined. The error was: 'openshift' is undefined

The error appears to have been in '/root/openshift-ansible/roles/openshift_master_facts/tasks/main.yml': line 31, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Set master facts
  ^ here
```
ansible-playbook playbooks/byo/openshift-node/scaleup.yml

For long output or logs, consider using a [gist](https://gist.github.com/)

##### Additional Information

Provide any additional information which may help us diagnose the
issue.

* Your operating system and version, ie: RHEL 7.2, Fedora 23 (`$ cat /etc/redhat-release`)
centos 7.3
* Your inventory file (especially any non-standard configuration parameters)
* Sample code, etc

```
EXTRA INFORMATION GOES HERE
```

**Comment by sdodson** [comment]
Can you try https://github.com/openshift/openshift-ansible/pull/5257 ?

**Comment by sdodson** [comment]
We've merged that pr can you try the latest from master?

**Comment by mtnbikenc** [comment]
I was seeing the same error as mentioned above which prompted me to (attempt to) fix the scaleup playbooks with #5257.  Please let me know if you are still seeing problems, or find new ones.  There has been quite a bit of refactoring that needs to be vetted through different code paths.

**Comment by aaryabhatt** [comment]
thanks guys, It solved the issue.
